chemdyg/templates/chemdyg_QBO_diags.bash

```diff
@@ -37,12 +37,8 @@ run_type="{{ run_type }}"
 obsDir="{{ diagnostics_base_path }}/observations/Atm/ChemDyg_inputs"
 ncfile_save="{{ ncfile_save }}"
 if [[ "${ncfile_save}" == "true" ]]; then
-   results_dir={{ output }}/post/atm/ncfiles
-   if [[ -d ${results_dir} ]]; then
-      echo "directory exists."
-   else
-      mkdir -p ${results_dir}
-   fi
+   results_dir={{ output }}/post/atm/chemdygfiles
```

Comment:
@hsiangheleellnl , can we make the directory name more meaningful? What about "chemdyg_output"? Or just "chemdyg" as Chris suggested?

Reply:
Okay. I changed it to chemdyg_output.